src/map/lua/lua_baseentity.cpp

```diff
@@ -4622,7 +4623,32 @@ inline int32 CLuaBaseEntity::checkNameFlags(lua_State* L)
         lua_pushboolean(L, false);
     return 1;
 }
+inline int32 CLuaBaseEntity::getLookSize(lua_State* L)
+{
+    TPZ_DEBUG_BREAK_IF(m_PBaseEntity == nullptr);
+
+    lua_pushinteger(L, m_PBaseEntity->GetLookSize());
+
+    return 1;
+}
 
+inline int32 CLuaBaseEntity::setLookSize(lua_State* L)
+{
+    TPZ_DEBUG_BREAK_IF(m_PBaseEntity == nullptr);
+
+    if (m_PBaseEntity->objtype == TYPE_PC)
```

Comment:
check not check for != TYPE_PC instead, and then avoid having the empty conditional?